packages/react/src/components/PageHeader/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef, ReactNode } from 'react';
+import SectionHeader from '../SectionHeader';
+import classNames from 'classnames';
+
+export interface PageHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
+  heading: ReactNode;
+  overline?: ReactNode;
+  description?: ReactNode;
+}
+
+const PageHeader = forwardRef<HTMLDivElement, PageHeaderProps>(
+  (
+    {
+      className,
+      heading,
+      overline,
+      description,
+      children,
+      ...otherProps
+    }: PageHeaderProps,
+    ref
+  ) => {
+    if (typeof heading === 'string') {
+      heading = <h1>{heading}</h1>;
+    }
+
+    return (
+      <div
+        className={classNames('PageHeader', className)}
+        ref={ref}
+        {...otherProps}
+      >
+        {overline && <p>{overline}</p>}
```

Comment:
Minor nit, `p` is typically used for paragraph text. In this instance I think a div with a class is more appropriate.

```suggestion
        {overline && <div className="PageHeader__overline">{overline}</div>}
```